### PR TITLE
patch: Inform users about RPi4 rev 1.5

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Contains the troubleshooting notice of RPi4 model B rev 1.5

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipul@balena.io>

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/faq/troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -68,10 +69,30 @@ If you are presented with a 'recovery login' prompt this usually indicates an is
...
 
 ### Wifi connection problems on RPI3
+
 If you are having issues with your RPI3 connecting to your wifi, make sure
 to check that the 2.4 GHz channel on your wifi router is set to something less than 11, since
 the firmware on the Raspberry Pi doesn't support channel 12 and 13.
 
+### balenaOS boot issues with new hardware revision of RPi4
+
+The Raspberry Pi Foundation recently released [hardware Rev 1.5](https://forums.raspberrypi.com/viewtopic.php?t=329299) of the Raspberry Pi 4 Model B. This revision of the board requires a relatively recent version of [balenaOS](https://balena.io/os). We recommend using balenaOS v2.88.5+rev1 or later containing the updated bootloader and firmware versions. BalenaOS v2.75.0+rev1 and earlier versions aren't tested and possibly won't boot on this revision of the board.
```
Comment (aallan) [inline pages/faq/troubleshooting/troubleshooting.md:79]: It wasn't the [Raspberry Pi Foundation](https://www.raspberrypi.org/), it was [Raspberry Pi](https://www.raspberrypi.com/). Raspberry Pi Ltd is the trading arm, we're the folks that produce the hardware, and maintain the software, and we're legally an entirely separate entity to the Foundation. There is no overlap in staffing or resources. We're both standalone entities.

Comment (vipulgupta2048) [reply pages/faq/troubleshooting/troubleshooting.md:79]: Thanks @aallan for catching this 👀 Appreciate you coming forward to rectify this. I will keep this in mind for the future and also inform the team of the same.

Comment (vipulgupta2048) [reply pages/faq/troubleshooting/troubleshooting.md]: Resolving..

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

File: pages/faq/troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -68,10 +69,30 @@ If you are presented with a 'recovery login' prompt this usually indicates an is
...
+```
+$ cat /proc/cpuinfo | grep Model 
+Raspberry Pi 4 Model B Rev 1.5
+```
+
+2. Look for the Dialog DA9090 power management controller next to the USB-C port. Check the picture below for reference.
+
+<img src="/img/troubleshooting/DA9090_identified.jpg" width="50%">
+
+For more information about Raspberry Pi 4 Model B Rev 1.5, please refer the Raspberry Pi’s [Product Information Portal](https://pip.raspberrypi.com/restricted).
```
Comment (lurch) [inline pages/faq/troubleshooting/troubleshooting.md:94]: "refer the ..." -> "refer to ..." ?

Comment (vipulgupta2048) [reply pages/faq/troubleshooting/troubleshooting.md:94]: Quite right, I will probably omit "Raspberry Pi's" too.

**Pull request merged**